runtime/near-vm/engine/src/universal/code_memory.rs

```diff
@@ -77,8 +99,10 @@ pub struct CodeMemory {
     /// Mapping size
     size: usize,
 
-    /// Size of the executable portion that comes first.
-    executable_size: usize,
+    /// Addresses `0..executable_end` contain executable memory.
+    ///
+    /// This is always be page-aligned.
```

Comment:
Nit: the data section must start page-aligned, we don’t have any code that makes `executable_end` itself be page-aligned